mediathread/juxtapose/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import User
+from courseaffils.models import Course
+from mediathread.djangosherd.models import SherdNote
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField()
+    spine = models.ForeignKey(SherdNote, blank=True, null=True)
+
+
+class JuxtaposeMediaItem(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    juxtaposition = models.ForeignKey(JuxtaposeAsset)
+    start_time = models.FloatField()
```

Comment:
I'm wondering if the inexact nature of the float type is going to cause weirdness when dealing with items that occur directly after each other, with no space in between. I will switch this to Django's [DecimalField](https://docs.djangoproject.com/en/1.10/ref/models/fields/#decimalfield) so we have the best control over these timestamps.